plugins/modules/win_certificate_store.ps1

```diff
@@ -262,10 +250,8 @@ Function New-CertFile($module, $cert, $path, $type, $password) {
         $cert_bytes = $file_encoding.GetBytes($cert_content)
     }
     elseif ($type -eq "pkcs12") {
-        $module.Result.key_exported = $false
-        if ($null -ne $cert.PrivateKey) {
-            $module.Result.key_exportable = $cert.PrivateKey.CspKeyContainerInfo.Exportable
-        }
+        # to ensure backwards compatibility, if $false Export will fail.
+        $module.Result.key_exportable = $true
```

Comment:
Why the change in this value? While it might still be `$false` for CNG keys it'll still be valid for CAPI ones. I prefer it to be sometimes right than always set to `$true`.